### ISSUE TITLE
[FEEDBACK]  Adding an introduction to npm installation packages

### ISSUE THREAD

**Issue by opfocus** [ethereum-optimism/docs#335]
Should `Optimism dev doc `consider adding an introduction to `npm `installation packages? This would make it easier for users to notice their presence and enhance package management.  https://www.npmjs.com/search?q=eth-optimism

**Comment by smartcontracts** [comment]
Which page do you think should be improved?

**Comment by opfocus** [comment]
The `Developer Tools `section seems like it could consolidate them. I noticed mentions of` @eth-optimism/contracts-bedrock, @eth-optimism/contracts-ts, @eth-optimism/sdk`, etc. in the documentation. Some others not mentioned include `@eth-optimism/web3.js-plugin`. I also came across this link: https://base.mirror.xyz/eUYUbxx-VHWVyZHMLyviKywHyIyOCq6kpbGhmreGQJA. I have a feeling that an advanced developer might not go through all the documentation, so consolidating them might be a good choice.
